Formula/cash-cli.rb

```diff
@@ -23,6 +23,6 @@ def install
   end
 
   test do
-    assert_match "Conversion of USD 100", shell_output("#{bin}/cash 100 USD PLN CHF")
+    assert_match "Something went wrong :(", shell_output("#{bin}/cash 100 USD PLN CHF", 1)
```

Comment:
```suggestion
    assert_match "Something went wrong :(", shell_output("#{bin}/cash 100 USD PLN CHF 2&>1", 1)
```
Whoops, forgot this earlier. If this is the test we want, anyway.

Reply:
With `2&>1` cash-cli just prints an empty string 🤔

Reply:
Well that's weird

Reply:
Ah, no it isn't. I messed up the redirection:
```suggestion
    assert_match "Something went wrong :(", shell_output("#{bin}/cash 100 USD PLN CHF 2>&1", 1)
```
I always forget the right syntax for this 😅

Reply:
Great, now I catches properly! 😅 Do you want me to fixup the commits into one?